amida-tech/greyscale
# Task/inba 292 task reducer mapping

#### What's this PR do?
Creates a function that retrieves a task with its corresponding flagged discussions
#### Related JIRA tickets:
[292](https://jira.amida-tech.com/browse/INBA-292)
#### How should this be manually tested?
Run the task.integration and discussion.integration tests to make sure the DB is populated with tasks and discussions:
- `node_modules/.bin/mocha test/discussion.integration.js --bail -t 10000`
- `node_modules/.bin/mocha test/taks.integration.js --bail -t 10000`

Hit this API end point using a taskId of 2:
`http://localhost:3005/testorg/v0.2/tasks-with-discussions/2`
Try it with a TaskId that has no discussions
`http://localhost:3005/testorg/v0.2/tasks-with-discussions/5`

#### Any background context you want to provide?
To simply return a task from the Tasks table, you can use the `selectOne` route:
http://localhost:3005/testorg/v0.2/tasks/2

#### Screenshots (if appropriate):
